x-pack/plugins/search_indices/public/components/start/start_page.tsx

```diff
@@ -45,7 +43,7 @@ export const ElasticsearchStartPage = () => {
         {isInitialLoading && <EuiLoadingLogo />}
         {hasIndicesStatusFetchError && <StartPageError error={indicesFetchError} />}
         {!isInitialLoading && !hasIndicesStatusFetchError && (
-          <ElasticsearchStart indicesData={indicesData} userPrivileges={userPrivileges} />
+          <ElasticsearchStart indicesData={indicesData} />
```

Comment:
@TattdCodeMonkey [ElasticsearchStart](https://github.com/elastic/kibana/blob/main/x-pack/plugins/search_indices/public/components/start/elasticsearch_start.tsx#L57) component  doesn't seem to be using `indicesData` prop. Can we replace this with `<ElasticsearchStart />`  ?